engine/src/java/com/runrev/android/nativecontrol/ScrollerControl.java

```diff
@@ -153,7 +153,10 @@ public boolean dispatchTouchEvent(MotionEvent e)
                     {
                         m_dragging = true;
                         m_tracking = false;
-                        onScrollBeginDrag();
+                        
+                        int x = (int)e.getX();
+                        int y = (int)e.getY();
+                        onScrollBeginDrag(x,y);
                         e.setAction(MotionEvent.ACTION_CANCEL);
```

Comment:
@livecodepanos can you check if these coordinates are relative to the top-left of the main view?

Reply:
Hmm, looks like they are not. I will investigate.